src/eko/kernels/evolution_integrals.py

```diff
@@ -40,6 +41,64 @@ def j00(a1, a0, nf):
     return np.log(a1 / a0) / beta.beta_qcd((2, 0), nf)
 
 
+@nb.njit(cache=True)
+def j00_qed(a1, a0, aem, nf):
```

Comment:
Maybe you can split all these new integrals in a new file `evolution_integrals_qed`, this way should be simpler to read and maintain.